### PR TITLE
[wip] Generate blockchain tests

### LINKED ISSUE #84: Fill Blockchain Tests

### PR DESCRIPTION
**Pull request by hugo-dc**
- Added wasm-tests branch to travis config (travis still needs to be activated for this repo)
- Changed the `Network` field from `ALL` to `Byzantium`.
- Generated Blockchain tests based from GeneralStateTests

Closes #84.

### CODE REVIEW

**Comment by axic** [comment]
Have you seen #107 ? :)

**Comment by hugo-dc** [comment]
Oh, sorry, i didn't see your PR, I think I'll only keep the Blockchain tests generation in this PR.

File: src/GeneralStateTestsFiller/stEWASMTests/wasmDivisionByZeroFiller.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ wasmDivisionByZero:
         gas:  !!int -1
         value: !!int -1
       network:
-        - ALL
+        - "Byzantium"
```
Comment (axic) [inline src/GeneralStateTestsFiller/stEWASMTests/wasmDivisionByZeroFiller.yml:42]: Shouldn't these use single quotes just like the rest of the fields?

**Comment by axic** [comment]
@hugo-dc @jwasinger can you please explain what benefit do we get with blockchain tests?

**Comment by hugo-dc** [comment]
@axic I'm not sure if the schema validator checks if there is Blockchain tests for ewasm or if everything is fine after the `bcEWASM` directory got removed. I'm ok to leave blockchain tests out if not needed by the schema validation.

**Comment by axic** [comment]
It only checks it because the directory exists (but is empty). @jwasinger said that for some tests blockchain tests are needed, though not sure which.

**Review by axic — CHANGES_REQUESTED** [review]
Please rebase over #107 once that is merged.

**Comment by hugo-dc** [comment]
I think this is not needed, I initially thought it would be needed for CI but is not the case.

Please reopen if you think it makes sense to have it.

**Pull request closed without merging**